packages/loopring_v3/contracts/core/impl/DefaultDepositContract.sol

```diff
@@ -92,9 +92,16 @@ contract DefaultDepositContract is IDepositContract, Claimable
         override
         payable
         onlyExchange
-        ifNotZero(amount)
+        // ifNotZero(amount)
```

Comment:
Maybe we just need to remove this line without adding the other code. @Brechtpd ?

Reply:
Yeah that seems to be sufficient to me.

Also, because we can't update the deposit contract, we can perhaps add a small additional check in `deposit` in `Exchangedeposits.sol`, but perhaps not that important:
```
uint16 tokenID = S.getTokenID(tokenAddress);
if (tokenID == 0 && amount == 0) {
     require(msg.value == 0), "INVALID_ETH_DEPOSIT");
}
```

Reply:
I added this logic